store/src/java-test/com/zimbra/cs/service/ResetPasswordTest.java

```diff
@@ -201,7 +201,7 @@ public void testResetPassword_FeatureDisabled() throws Exception {
 
 class TestResetPassword extends ResetPassword {
     @Override
-    protected void setPasswordAndPurgeAuthTokens(Provisioning prov, Account acct, String newPassword)
+    protected void setPasswordAndPurgeAuthTokens(Provisioning prov, Account acct, String newPassword, boolean dryRun)
```

Comment:
This is creating issue in build creation, we could bypass running unit tests when creating build to solve this problem but I would not prefer it
```
test-compile:
    [mkdir] Created dir: /root/jenkins/workspace/ZWC_Build/zm-mailbox/store/build/test-classes
    [javac] Compiling 278 source files to /root/jenkins/workspace/ZWC_Build/zm-mailbox/store/build/test-classes
    [javac] /root/jenkins/workspace/ZWC_Build/zm-mailbox/store/src/java-test/com/zimbra/cs/service/ResetPasswordTest.java:203: error: method does not override or implement a method from a supertype
    [javac]     @Override
    [javac]     ^
    [javac] Note: Some input files use or override a deprecated API.
    [javac] Note: Recompile with -Xlint:deprecation for details.
    [javac] Note: Some input files use unchecked or unsafe operations.
    [javac] Note: Recompile with -Xlint:unchecked for details.
    [javac] 1 error
```

Reply:
The build issue is fixed after this commit.